apps/functions-subscription/src/main.ts

```diff
@@ -1,10 +1,10 @@
 import { app } from '@azure/functions';
-import { InfoFunction } from './adapters/azure/functions/info';
+import { makeInfoHandler } from './adapters/azure/functions/info';
 
 // eslint-disable-next-line functional/no-expression-statements
 app.http('info', {
   methods: ['GET'],
   authLevel: 'anonymous',
-  handler: InfoFunction({}),
+  handler: makeInfoHandler({}),
```

Comment:
**note**: I have renamed it to be more aligned with the name (`handler`) of `app.http` inputs